src/views/catalog/templatescatalog/components/TemplatesCatalogDrawer/TemplatesCatalogDrawerCreateForm.tsx

```diff
@@ -43,13 +50,22 @@ export const TemplatesCatalogDrawerCreateForm: React.FC<TemplatesCatalogDrawerCr
   React.memo(({ namespace, template, canQuickCreate, isBootSourceAvailable, onCancel }) => {
     const history = useHistory();
     const { t } = useKubevirtTranslation();
-    const [vmName, setVmName] = React.useState(generateVMName(template));
+    const { updateVM } = useWizardVMContext();
+    const { ns } = useParams<{ ns: string }>();
+
+    const [vmName, setVmName] = React.useState('');
```

Comment:
nitpick: It should be `setVMName`. All letters in an abbreviation should be capitalized.